src/assets/source/scripts/modules/serlo_breadcrumbs.js

```diff
@@ -0,0 +1,181 @@
+/**
+ *
+ * Athene2 - Advanced Learning Resources Manager
+ *
+ * @author  Jonas Keinholz (jonas.keinholz@serlo.org)
+ * @license LGPL-3.0
+ * @license http://opensource.org/licenses/LGPL-3.0 The GNU Lesser General Public License, version 3.0
+ * @link    https://github.com/serlo-org/athene2 for the canonical source repository
+ *
+ * Breadcrumb
+ *
+ */
+
+/*global define*/
+define('breadcrumbs', ['jquery'], function ($) {
+    'use strict';
+    var Breadcrumbs,
+        instance,
+        defaults;
+
+    defaults = {
+        // main wrapper selector
+        wrapperId: '#subject-nav-wrapper',
+
+        // breadcrumb selector
+        breadcrumbId: '#breadcrumbs',
+
+        // desired height
+        height: '45'
+    };
+
+    /**
+     * @class Breadcrumbs
+     * @param {Object} options See defaults
+     *
+     * Main constructor
+     **/
+    Breadcrumbs = function (options) {
+        if (!(this instanceof Breadcrumbs)) {
```

Comment:
No idea, but there could be a conflict when using this funciton in a callback without using `.bind(this)`. Probably keep it in there just to be save.